scripts/geoJSON.js

```diff
@@ -0,0 +1,49 @@
+var config = require('./config.js')
+var coords = require('./coords.js')
```

Comment:
I like the idea of tidier code, but I'm not following sorry! trying to figure out how to piece things together.  . . so in `index.js` would I go 
`var markerPoints = geoJSON.(createGeoJSON)`
`renderMap(markerPoints)`